tests/constants.py

```diff
@@ -24,6 +24,7 @@
 TEST_GIT_URI = "git://hostname/path"
 TEST_GIT_URI_HUMAN_NAME = "path"
 TEST_GIT_REF = "0123456789012345678901234567890123456789"
+TEST_GIT_URI_HTML = "git%3A%2F%2Fhostname%2Fpath"
```

Comment:
Or, could use this:
```
from six.moves.urllib.parse import quote as urlquote
urlquote(TEST_GIT_URI, safe='')
```

Reply:
Fixed.

Reply:
Seems like that causes problems on CentOS 6 with Python 2.6, not sure why. I'm fine with putting it back how it was, although it might be better if it was the line next to the definition of TEST_GIT_URI.

Reply:
Fixed again, along with a comment as to why the code isn't using the urlparse lib